src/GoogleMapField.php

```diff
@@ -9,6 +9,18 @@
  * Maps Geocoding API.
  * @author <@willmorgan>
  */
+
+namespace BetterBrief\GoogleMapField;
```

Comment:
I think the namespace should just be `BetterBrief` so we can do `new BetterBrief\GoogleMapField` rather than `new BetterBrief \GoogleMapField\GoogleMapField` which is clunky